lib/firmata.js

```diff
@@ -741,6 +750,14 @@ Board.prototype.sendString = function(string) {
   this.sp.write(data);
 };
 
+function i2cRequest(board, buffer) {
+
+  if (!i2cActive.get(board)) {
```

Comment:
An alternative is to call `i2cConfig(0)` instead of throwing an error. If non-firmata I2C implementations don't have a concept of configuration or initialization and you simply read or write, then forcing a call to config from the J5 level may be problematic in the future. However I'd suspect most if not all implementations would have some sort of an init and some architectures may even require parameters that Arduino does not (such as high speed I2C). In that case forcing config makes sense... tough call.